script.js

```diff
@@ -74,3 +79,31 @@ removeButton.forEach((remove) => {
     document.location.reload();
   });
 });
+
+// nav bar links functionality
+
+const list = document.querySelector('#list');
+const addNew = document.querySelector('#navAdd');
+const contact = document.querySelector('#navContact');
+const addBook = document.querySelector('#add-book');
+const contactForm = document.querySelector('#contact');
+const bookSection = document.querySelector('#book-section');
+
+list.addEventListener('click', () => {
+  addBook.classList.remove('active');
+  contactForm.classList.remove('active');
+  bookSection.classList.remove('inactive');
+  document.location.reload();
+});
+
+addNew.addEventListener('click', () => {
+  addBook.classList.add('active');
+  bookSection.classList.add('inactive');
+  contactForm.classList.remove('active');
+});
+
+contact.addEventListener('click', () => {
+  contactForm.classList.add('active');
+  bookSection.classList.add('inactive');
+  addBook.classList.remove('active');
+});
```

Comment:
- [ ]  You should display a page relative to its navigation. I am trying to click on specific navigation but the page won't change. fix this issue so that users can be able to see the add new and contact page 
![image](https://user-images.githubusercontent.com/43877689/177599761-fb9b53f8-122f-422e-92a6-2e0d49137424.png)

- [ ]  Make sure that your website has Three content sections - Books list, Add book form, and Contact info. and all those sections should resemble the one provided in the reference design.